README.es.md

```diff
@@ -0,0 +1,135 @@
+
+Para probar como puede ser usada tu definición cuando se refieren globalmente o como un módulo importado, crea una carpeta `test`, y coloca dos archivos de prueba en el.  Name one `YourLibraryName-global.test.ts` and the other `YourLibraryName-module.test.ts`.  The *global* test file should exercise the definition according to how it would be used in a script loaded on a web page where the library is available on the global scope - in this scenario you should not specify an import statement.  The *module* test file should exercise the definition according to how it would be used when imported (including the `import` statement(s)).  If you specify a `files` property in your `tsconfig.json` file, be sure to include both test files.  A [practical example of this](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/big.js/test) is also available on the big.js definition.
+
+Porfavor tenga en cuenta que no es necesario para ejercer plenamente la definición en cada archivo de prueba - Es suficiente con probar solo los elementos globalmente accesibles en la prueba de archivos globar y ejercer la definición en el módulo del archivo de prueba, o viceversa.
+
+#### ¿Qué pasa con paquetes scoped?
+
+Types para un paquete scoped `@foo/bar` deberán ir en `types/foo__bar`. tenga en cuenta el doble guión bajo.
+
+Cuando `dts-gen` es utilizado como scaffold en un paquete scoped, las propiedades `paths` deberan ser adaptadas manualmente en el paquete generado
+`tsconfig.json` para referenciar correctamente el paquete scoped:
+
+```json
+{
+    "paths":{
+      "@foo/bar": ["foo__bar"]
+    }
+}
+```
+
+
+#### El historial de archivos en GitHub parece incompleto.
+
+GitHub no le hace [support](http://stackoverflow.com/questions/5646174/how-to-make-github-follow-directory-history-after-renames) historial de archivos para archivos renombrados. Utilice [`git log --follow`](https://www.git-scm.com/docs/git-log) en su lugar.
+
+#### Debería añadir un espacio de nombres vacío que no exporte un módulo que utilice ES6 style imports?
+
+Algunos paquetes, como [chai-http](https://github.com/chaijs/chai-http), exportan una función.
+
+importar este módulo con un ES6 style import de forma `import * como foo de "foo";` conduce al error:
+
+> error TS2497: El módulo 'foo' resuelve a una entidad non-module y no puede ser importado utilizando esta construcción
```

Comment:
Use the right translated message:

error ts2497: El módulo 'foo' se resuelve en una entidad que no es un módulo y no se puede importar mediante esta construcción